deployments/warp_routes/sonicSOL/solanamainnet-sonicsvm-config.yaml

```diff
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=../schema.json
+tokens:
+  - addressOrDenom: Fhqa36L7sFJDiMdu54WUah9stAwRacycqHJYm3b3Geje
+    chainName: solanamainnet
+    collateralAddressOrDenom: sonickAJFiVLcYXx25X9vpF293udaWqDMUCiGtk7dg2
```

Comment:
is this a valid address

Reply:
hah yes it's a vanity address they generated https://solscan.io/token/sonickAJFiVLcYXx25X9vpF293udaWqDMUCiGtk7dg2